unitTests/ui/QuerySummaryTest.ts

```diff
@@ -307,16 +308,16 @@ export function QuerySummaryTest() {
       });
 
       it(`when the query searched is an empty string,
-          it should replace the query tag with the query searched`, () => {
+          it should put the string NoResult instead`, () => {
         test = Mock.optionsComponentSetup<QuerySummary, IQuerySummaryOptions>(QuerySummary, {
           enableNoResultsFoundMessage: true,
-          noResultsFoundMessage: `${queryTag}`
+          noResultsFoundMessage: `customMessage ${queryTag}`
```

Comment:
Not sure I understand what the `customMessage` does. Can you explain? 🤔

Reply:
It's not really necessary, I saw it was used in the other tests and found it more realistic for the use case.

Reply:
If the query is "test", the ouput would be "customMessage test". If the query is empty, it should default to "No result"